### PR TITLE
Assignment 1.2

### PR DESCRIPTION
**Pull request by SkymeFactor**
Accuracy 0.194, linear classifier

### CODE REVIEW

**Comment by SkymeFactor** [comment]
Оч торопился, поэтому график пляшет, уверен, если подтянуть код, можно было бы поднять до 25%

**Comment by SkymeFactor** [comment]
Изменил некоторые параметры, получил в итоге accuracy 0,206

File: Vlad_Sergei(Skyme)/Assignment1.2/linear_classifer.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+      zeros[target_index] = 1
+
+    # print(sumlog.shape, x_i.shape)
+    loss = sumlog - x_i.T
+    grad = softmax(predictions)
+    grad -= zeros
+
+    if predictions.ndim > 1:
+      grad /= predictions.shape[0]
+    return loss.mean(), grad
```
Comment (toshiks) [inline Vlad_Sergei(Skyme)/Assignment1.2/linear_classifer.py]: Перепиши это таким образом, чтобы ты использовал написанные тобой softmax и cross_entropy_loss
